src/components/security_manager/test/security_manager_test.cc

```diff
@@ -87,6 +89,8 @@ uint8_t handshake_data_out[] = {0x6, 0x7, 0x8};
 uint8_t* handshake_data_out_pointer = handshake_data_out;
 const size_t handshake_data_out_size =
     sizeof(handshake_data_out) / sizeof(handshake_data_out[0]);
+const uint32_t kAsyncExpectationsTimeout = 10000u;
```

Comment:
@orhan-mehmedov Generally speaking you are right and I can fix this particular one, but what about all others timeout constants? Probably the right way is to follow some common guidelines in naming conversions and apply them to the newly added code.